.github/workflows/wheels.yml

```diff
@@ -195,12 +195,16 @@ jobs:
         name: dist
         path: dist/*.tar.gz
 
-  success:
-    permissions:
-      contents: none
+  pypi-publish:
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
     needs: [build, windows, sdist]
     runs-on: ubuntu-latest
-    name: Wheels Successful
+    name: Upload release to PyPI
```

Comment:
We'll need a step or two before this to download the artifacts into the correct dir using https://github.com/actions/download-artifact, so we have the files from the preceding jobs.

Reply:
I've added a step to the download the artifacts. I presume they should all be together, so I also added a step to move them out of their individual directories (now that actions/upload-artifact v4 means that artifacts need different names).